newton/examples/diffsim/example_diffsim_spring_cage.py

```
@@ -157,17 +159,50 @@ def forward(self):
+            x_1 = x_c + step
+            x_0 = x_c - step
+            param.assign(x_1)
+            l_1 = self.forward().numpy()[0]
+            param.assign(x_0)
+            l_0 = self.forward().numpy()[0]
+            dldx = (l_1 - l_0) / (eps * 2.0)
+            x_grad_numeric[i] = dldx
+        param.assign(x_c)
+        tape = wp.Tape()
+        with tape:
+            l = self.forward()
```

Ruff (0.13.1)

Ambiguous variable name: `l`

(E741)